hack/build-rpms.sh

```diff
@@ -72,16 +74,19 @@ else
 	mkdir -p "${rpm_tmp_dir}/SOURCES"
 	tar czf "${rpm_tmp_dir}/SOURCES/${OS_RPM_NAME}-${OS_RPM_VERSION}.tar.gz" \
 		--owner=0 --group=0 \
-		--exclude=_output --exclude=.git --transform "s|^|${OS_RPM_NAME}-${OS_RPM_VERSION}/|rSH" \
+		--exclude=_output --exclude=.git \
+		--transform "s|^|${OS_RPM_NAME}-${OS_RPM_VERSION}/|rSH" \
 		.
 
 	rpmbuild -b${srpm} "${OS_RPM_SPECFILE}" \
-		--define "skip_dist 1" \
+		--define "skip_dist ${SKIP_DIST:-1}" \
+		--define "ose_build ${OSE_BUILD:-1}" \
 		--define "make_redistributable ${make_redistributable}" \
-		--define "version ${OS_RPM_VERSION}" --define "release ${OS_RPM_RELEASE}" \
+		--define "version ${OS_RPM_VERSION}" \
+		--define "release ${OS_RPM_RELEASE}" \
 		--define "commit ${OS_GIT_COMMIT}" \
 		--define "os_git_vars ${OS_RPM_GIT_VARS}" \
-		--define 'dist .el7' --define "_topdir ${rpm_tmp_dir}"
```

Comment:
can I safely remove the `dist` override here? Looks like it it is skipped anyway by `skip_dist`